packages/@webex/plugin-cc/src/services/config/index.ts

```diff
@@ -97,36 +176,195 @@ export default class AgentConfigService {
         throw new Error(`API call failed with ${response.statusCode}`);
       }
 
-      this.webex.logger.log('getListOfTeams api success.');
+      LoggerProxy.logger.log('getListOfTeams api success.');
 
       return Promise.resolve(response.body);
     } catch (error) {
-      return Promise.reject(error);
+      LoggerProxy.logger.error(`getListOfTeams API call failed with ${error}`);
+      throw error;
     }
   }
 
-  /**
-   * Method to get List of AuxCodes.
-   * @param {number} page Index of the page of results to be fetched. Defaults to 0.
-   * @param {number} pageSize Number of items to be displayed on a page. Defaults to 10.
-   * @param {Array<String>} filter Filter that can be applied to the elements to be fetched. Defaults to [].
-   * @param {Array<String>} attributes Specify the attributes to be returned. Defaults to ['id', 'name', 'active'].
-   * @returns {Promise<ListAuxCodesResponse>} A promise that eventually resolves to an API response.
-   */
+  public async getAllTeams(
```

Comment:
If possible, can we get a brief description about it.

Reply:
addressed